blast_config/tx_struct_tng.c

```diff
@@ -1042,8 +883,26 @@ channel_t channel_list[] =
     // Fields to be added here
 
     /*---------------------------------------------------------------------------------------*/
-    /*                                        MCP Misc                                       */
+    /*                                        MCP core                                       */
     /*---------------------------------------------------------------------------------------*/
+
+    // MCP core information
+    { "time",                 SCALE(CONVERT_UNITY), TYPE_UINT32, RATE_100HZ, U_NONE, 0 },
+    { "time_usec",            SCALE(CONVERT_UNITY), TYPE_UINT32, RATE_100HZ, U_NONE, 0 },
+    { "parts_sched",          SCALE(CONVERT_UNITY), TYPE_UINT32, RATE_100HZ, U_NONE, 0 },
+    { "status_mcc",           SCALE(CONVERT_UNITY), TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "upslot_sched",         SCALE(CONVERT_UNITY), TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "veto_sensor",          SCALE(CONVERT_UNITY), TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+
+    // Telemetry
+    { "plover",               SCALE(CONVERT_UNITY), TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
```

Comment:
Is this a magic word? Comment to explain.

Reply:
added